Mathlib/AlgebraicTopology/SimplicialSet/Path.lean

```diff
@@ -85,4 +85,47 @@ lemma Path.ext' {n : ℕ} {f g : Path X (n + 1)}
       rw [← f.arrow_tgt (Fin.last n), ← g.arrow_tgt (Fin.last n), h]
   · exact h j
 
+/-- Maps of simplicial sets induce maps of paths in a simplicial set.-/
+@[simps]
+def mapPath {X Y : SSet.{u}} {n : ℕ} (σ : X ⟶ Y) (f : X.Path n) : Y.Path n where
+  vertex i := σ.app (Opposite.op [0]) (f.vertex i)
+  arrow i := σ.app (Opposite.op [1]) (f.arrow i)
+  arrow_src i := by
+    simp only [← f.arrow_src i]
+    exact (congr (σ.naturality (δ 1).op) (by rfl)).symm
+  arrow_tgt i := by
+    simp only [← f.arrow_tgt i]
+    exact (congr (σ.naturality (δ 0).op) (by rfl)).symm
+
+/-- `mapPath` respects subintervals of paths.-/
+lemma mapPath_interval {X Y : SSet.{u}} {n : ℕ} (σ : X ⟶ Y) (f : X.Path n)
+    (j l : ℕ) (hjl : j + l ≤ n) :
+    (mapPath σ f).interval j l hjl = mapPath σ (f.interval j l hjl) := rfl
+
+/-- The spine of the unique non-degenerate `n`-simplex in `Δ[n]`.-/
+def idSpine (n : ℕ) : Path Δ[n] n := spine Δ[n] n (standardSimplex.idSimplex n)
```

Comment:
This should be in the `standardSimplex` namespace, probably as `spineId`.

Reply:
In general, does recommending moving something to another namespace imply that the definition should be moved to the file in which that namespace is defined? Or should it just be added to the namespace from its current location?

The ability to freely append to a namespace in Lean is still somewhat foreign to me.

Reply:
In Lean, filenames and namespaces are independent. Here, this makes sense in the file `SimplicalSet.Path`, but it would be better in the `standardSimplex`, and it does not need to be moved to another file.